applications.rst

```diff
@@ -27,3 +27,4 @@ Applications
    fsm
    planning
    general
+   email_marketing
```

Comment:
general should be kept as the last page.
Also, don't you think that it'd be better to group email_marketing with social_marketing and sms_marketing ?

Reply:
that is what I was trying to do.